### ISSUE TITLE
Intake | Remove non-existent contention error

### ISSUE THREAD

**Issue by pkarman** [department-of-veterans-affairs/caseflow#9800]
See https://dsva.slack.com/archives/CCB97HZJL/p1551815313002300

We get a 'NoMethodError` Ruby exception when we attempt to remove a contention that does not exist.

There are multiple ways forward on this and it is unclear why this is happening.
 
It might be possible to edit a claim review multiple times and queue multiple RequestIssuesUpdate jobs to remove the same contention.

We should flag issues that have been queued for removal so that users cannot "remove" them twice.

**Comment by pkarman** [comment]
One example:

A SC was created. An unidentified request issue was removed and another added. Then the 2nd unidentified issue was replaced with a rating request issue.

```
[#<RequestIssuesUpdate:0x00005596813d1068
  id: 132,
  user_id: 5144,
  review_type: "SupplementalClaim",
  review_id: 802,
  before_request_issue_ids: [7453],
  after_request_issue_ids: [9671],
  processed_at: Fri, 15 Mar 2019 19:39:56 UTC +00:00,
  attempted_at: Fri, 15 Mar 2019 19:39:50 UTC +00:00,
  submitted_at: Fri, 15 Mar 2019 19:39:50 UTC +00:00,
  error: nil,
  last_submitted_at: Fri, 15 Mar 2019 19:39:50 UTC +00:00>,
 #<RequestIssuesUpdate:0x00005596813d0f28
  id: 133,
  user_id: 5144,
  review_type: "SupplementalClaim",
  review_id: 802,
  before_request_issue_ids: [9671],
  after_request_issue_ids: [9673],
  processed_at: nil,
  attempted_at: Mon, 18 Mar 2019 19:52:59 UTC +00:00,
  submitted_at: Fri, 15 Mar 2019 19:54:37 UTC +00:00,
  error: nil,
  last_submitted_at: Fri, 15 Mar 2019 19:54:37 UTC +00:00>]
```

The 2nd unidentified request issue (9671) re-used the removed contention id assigned to the 1st unidentified request issue (7453). Then when we attempted to remove 9671, because it was pointing to a non-existent contention, we get the error.

**Comment by pkarman** [comment]
fixed by #10041
